next.config.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```

Comment:
remove the /* eslint-disable */ in this page. And fix the indents on this file, we use 4 spaces or 2 tabs. You are doing more than that here